sotodlib/tod_ops/jumps.py

```diff
@@ -4,14 +4,18 @@
 import scipy.ndimage as simg
 import scipy.stats as ss
 from numpy.typing import NDArray
-from pixell.utils import block_expand, block_reduce, moveaxis
+from pixell.utils import moveaxis
 from scipy.sparse import csr_array
 from skimage.restoration import denoise_tv_chambolle
 from so3g import (
-    matched_jumps,
-    matched_jumps64,
+    block_minmax,
```

Comment:
One way to create some backwards-compatibility is to not import these by name.  Just import so3g.  Then the code will crash when you try to use those functions, rather than when you try to import them.

As written now, everyone's installation will break, if they try to use _anything_ from `sotodlib.tod_ops`, unless they've updated so3g.